uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-assignments/modules/tasks-assignments.settings.config.js

```diff
@@ -25,12 +28,15 @@ define([
         templateUrl: config.path.TPL + 'settings.html?v=5'
       });
 
-    $resourceProvider.defaults.stripTrailingSlashes = false;
+    $analyticsProvider.withAutoBase(true);
+    $analyticsProvider.settings.ga = {
```

Comment:
An empty line is needed above.

Reply:
you mean like this?
```js
$analyticsProvider.withAutoBase(true);

$analyticsProvider.settings.ga = {
```
If yes, then I'd rather keep it like this, given that it groups the `$analyticsProvider` lines together

Reply:
If you wrap this into a function, you won't need grouping. Please follow the styleguide.